Tests/Functional/Scripts/requireAdditionalVendors.sh

```diff
@@ -8,7 +8,9 @@ CURRENT_REF=${GITHUB_HEAD_REF:-$GITHUB_REF}
 CURRENT_BRANCH=${CURRENT_REF#refs/heads/}
 if [ "$CURRENT_BRANCH" != "master" ]; then
     composer config minimum-stability dev
+    composer config repositories.fork vcs ${SERVER_URL}/${REPOSITORY}.git
```

Comment:
This really needs to be the environment variable names I proposed, because they will be set automatically in GitHub actions:
```suggestion
    composer config repositories.fork vcs ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git
```

Reply:
Thank you, I am quite stucked with the breaklines lines for the image here

I tried to revert by using `git checkout 2cbf03e -- doc/img/esi-trace.png`
```
warning: CRLF will be replaced by LF in doc/img/esi-trace.png.
The file will have its original line endings in your working directory
```
With this warning and when I try to add them my github automatically reverted to my current state with no changes . How do you come with this kind of issue specially when there is multiple os involved?